### PR TITLE
Added more instructions for running the operator

### PR DESCRIPTION
**Pull request by arjunrn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +44,20 @@ in a OpenShift cluster.
     --from-file=credentials=credentials
     ```
    Alternatively use the `CredentialRequest` resource in the `hack` directory
```
Comment (thejasn) [inline README.md:46]:
```suggestion
   Alternatively use the `CredentialsRequest` resource in the `hack` directory
```

File: README.md [hunk 2]
```diff
@@ -4,14 +4,35 @@ This operator is used to install, manage and configure an instance of
...
 ### Running the operator
 
-1. Build and push the image to an image registry.
+1. Replace the operand image in the file `config/manager/manager.yaml` in 
+   the environment variable `RELATED_IMAGE_CONTROLLER` with the image 
+   created in the previous step.
+ 
+2. Build and push the image to an image registry.
     ```bash
     export IMG=example.com/aws-load-balancer-operator:latest
```
Comment (lihongan) [inline README.md:32]: To follow the similar style with operand image, can we update this as 
```
export IMG=quay.io/$USER/aws-load-balancer-operator:latest
```

Comment (arjunrn) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -4,14 +4,35 @@ This operator is used to install, manage and configure an instance of
...
+   created in the previous step.
+ 
+2. Build and push the image to an image registry.
     ```bash
-    export IMG=example.com/aws-load-balancer-operator:latest
+    export IMG=quay.io/$USER/aws-load-balancer-operator:latest
     make image-build image-push
     ```
-2. Create AWS credentials profile for the operator
+3. Create AWS credentials profile for the operator
```
Comment (alebedev87) [inline README.md:35]: What I noticed trying these instructions is that the secret or credentials request is created in the operator namespace which doesn't exist yet as it'll be created in the last command: `make deploy`.

**Comment by lihongan** [comment]
Following the doc and we can deploy the operator and test-e2e is passed.
```
$ oc -n aws-load-balancer-operator get pod
NAME                                                            READY   STATUS    RESTARTS   AGE
aws-load-balancer-controller-cluster-697575547-79bql            1/1     Running   0          51s
aws-load-balancer-operator-controller-manager-64cd5fb94-gb4l8   2/2     Running   0          2m6s
```

**Pull request merged**